Source/Charts/Charts/RadarChartView.swift

```diff
@@ -168,21 +168,9 @@ open class RadarChartView: PieRadarChartViewBase
         let sliceAngle = self.sliceAngle
         
         let max = _data?.maxEntryCountSet?.entryCount ?? 0
-        
-        var index = 0
-        
-        for i in 0..<max
-        {
-            let referenceAngle = sliceAngle * CGFloat(i + 1) - sliceAngle / 2.0
-            
-            if referenceAngle > a
-            {
-                index = i
-                break
-            }
-        }
-        
-        return index
+        return (0..<max).firstIndex {
+            sliceAngle * CGFloat($0 + 1) - sliceAngle / 2.0 > a
+        } ?? max
```

Comment:
@jjatie while I am working merging 4.0, I found this imp is different than the old one.

The old one would return 0 if no match, but the new logic returns max.

Is this a mistake or intended?

Reply:
It's the same logic.

Reply:
My mistake. Should be `?? 0`
It might be a bit more readable to write it as:
```
return (0..<max).firstIndex {
    let referenceAngle = sliceAngle * CGFloat($0 + 1) - sliceAngle / 2.0
    return referenceAngle > a
} ?? 0
```

Reply:
I have changed to ?? 0.
I will let you know when I resolve all conflicts and we could continue from there.